api/base/serializers.py

```diff
@@ -44,6 +44,48 @@ def format_relationship_links(related_link=None, self_link=None, rel_meta=None,
     return ret
 
 
+class HideIfRetraction(ser.Field):
+    """
+    If node is retracted, this field will return None.
```

Comment:
Yes, I discussed this with @brianjgeiger. I was originally hiding the fields, but it was thought that it was better to return null since the user would be expecting the field.